File: Easy/Recursive sequence.py
#User function Template for python3

class Solution:
    def sequence(self, n):
        # code here
        m = 1000000007
        ans = 0
        
        temp = 1
        
        for i in range(1, n + 1):
            t = 1
            
            for j in range(i):
                #print(temp, end = " ")
                t = (t * temp) % m
                temp += 1
            
            #print("\n", t)
            ans = (ans + t) % m
        
        return ans % m

#{ 
 # Driver Code Starts
#Initial Template for Python 3

if __name__ == '__main__':
    t = int(input())
    for _ in range(t):
        N = int(input())
        
        ob = Solution()
        print(ob.sequence(N))
# } Driver Code Ends

File: Easy/Recursive sequence.cpp
//{ Driver Code Starts
//Initial Template for C++

#include <bits/stdc++.h>
using namespace std;

// } Driver Code Ends
//User function Template for C++

class Solution{
public:
    long long sequence(int N)
    {
        // code here
        long long ans = 0;
        long long conta = 1;
        
        for (int i=0; i<N; i++)        
        {
            long long temp = 1;
            for (int j=0; j<i+1; j++)
            {
                //cout << conta << " ";
                temp *= conta;
                conta++;
            }
            //cout << endl;
            ans += temp;
        }
        
        return ans;
        
    }
};

//{ Driver Code Starts.

int main(){
    int t;
    cin>>t;
    while(t--){
        int N;
        cin>>N;
        
        Solution ob;
        cout<<ob.sequence(N)<<endl;
    }
    return 0;
}
// } Driver Code Ends